src/dxt.rs

```diff
@@ -18,7 +18,7 @@ use crate::image::{self, ImageDecoder, ImageDecoderExt, ImageReadBuffer, Progres
 /// Note that DXT2 and DXT4 are left away as they're
 /// just DXT3 and DXT5 with premultiplied alpha
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
-pub enum DXTVariant {
+pub enum DxtVariant {
```

Comment:
So have missed something somewhat important in the description of the issue, and sadly did not check this sooner. The ability to use enum variants through type aliases got introduced [in `1.37`](https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1370-2019-08-15). Since our MSRV policy is `1.34.2` support this would technically be a breaking change to rename enums. I think we sadly have to keep the orginal names for now for enums, and the alias can have the new capitalization. Clarification: The struct types should be fine.

Reply:
I completely forgot to change my rustc version, sorry about that.

I can change that.

Reply:
I also changed `PnmSubtype`. I can fix that too.

Reply:
Please do, yes. And add a `TODO` comment referencing either the fact or the comment.